centreon/src/Core/Dashboard/Playlist/Infrastructure/API/CreatePlaylist/CreatePlaylistRoute.yaml

```diff
@@ -0,0 +1,6 @@
+CreatePlaylist:
+  methods: POST
+  path: /configuration/dashboards/playlists
+  requirements:
+  controller: 'Core\Dashboard\Playlist\Infrastructure\API\CreatePlaylist\CreatePlaylistController'
+  condition: "request.attributes.get('version') >= 23.10"
```

Comment:
>= 23.10 ? not 24.04 ? It won't be backported, isn't it ?

Reply:
It will, Dashboard are in beta, so everything is backported

Reply:
🤯